posts/2021/weather-station-data-preprocess.ipynb

```diff
@@ -0,0 +1,1934 @@
+{
```

Comment:
<p><code>Dealing the time</code> -&gt; <code>Dealing with the time</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>